apps/explorer/src/context.ts

```diff
@@ -37,14 +37,17 @@ export function useNetwork(): [string, (network: Network | string) => void] {
     }, [searchParams]);
 
     const setNetwork = (network: Network | string) => {
-        // When resetting the network, we reset the query client at the same time:
-        queryClient.cancelQueries();
-        queryClient.clear();
-
         setSearchParams({ network: network.toLowerCase() });
     };
 
-    useLayoutEffect(() => {
```

Comment:
I don't think this should be a layout effect... but correct me if I'm missing something @Jordan-Mysten